src/main/java/com/conveyal/datatools/manager/gtfsplus/CalendarAttribute.java

```diff
@@ -0,0 +1,20 @@
+package com.conveyal.datatools.manager.gtfsplus;
+
+import com.conveyal.gtfs.model.Entity;
+
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+public class CalendarAttribute extends Entity {
```

Comment:
Needs javadoc

Reply:
Same as above comment: I'm not really sure what I would say for each of these tables. We don't really have javadoc for these entities on the GTFS side of things, so I wonder why we would need it here.